### PR TITLE
[backend] API and API docs

### PR DESCRIPTION
**Pull request by Error0229**
Any issue in PR #12  should already resolved 😶‍🌫️
And the Swagger UI endpoint would only open when the environment variable `TWP_ENV = dev` 
So, export that before run the echo server

Run the server and visit [`localhost:8080/swagger/`]() (remember the last slash) to check out Swagger UI

Want to try?
```
go install github.com/swaggo/swag/cmd/swag@latest
```
Follow the [guide](https://github.com/swaggo/swag#declarative-comments-format) to add/modify annotation for API code
```
swag init -g pkg/router/router.go
```
Restart the server then the API docs will be there 📃

### CODE REVIEW

**Review by UGisBusy — DISMISSED** [review]
Works as intended. Can access to swagger ui if no frontend.
The detail still need to wait for noobdy tho.
But hey it lgtm.

File: main.go [hunk 1]
```diff
@@ -18,7 +18,10 @@ func main() {
 
 	e.Use(middleware.Static("frontend/dist"))
 
-	pkg.RouterHandler(e.Group("/api"))
+	router.RegisterApi(e)
+	if os.Getenv("TWP_ENV") == "dev" {
```
Comment (NOOBDY) [inline main.go]: it is better to make `"dev"` into a const or something for better readability and prevent typos
example:
```go
type Environment string

const (
    DEV Environment = "dev"
    PROD Environment = "prod"
)
```

**Review by NOOBDY — CHANGES_REQUESTED** [review]
I'm too lazy to examine every single endpoint, I believe problems will show themselves when the timing is correct

File: main.go (hunk 1 above)
Comment (Error0229) [reply main.go]: + Add new package environment for env variables

**Review by NOOBDY — APPROVED** [review]
LGTM

**Review by UGisBusy — APPROVED** [review]
LGTM
Can comfirm it still works after changes.

**Pull request merged**